content/blog/command-provider-ga/index.md

```diff
@@ -0,0 +1,221 @@
+    // Replace the resource, i.e., re-run the 'create' command,
+    // when the pet resource changes.
+    triggers: [pet.id],
+});
+```
+
+{{% /choosable %}}
+
+{{% choosable language python %}}
+
+```python
+import pulumi
+import pulumi_command as command
+import pulumi_random as random
+
+pet = random.RandomPet("pet")
+
+pet_file_command = command.local.Command("petFileCommand",
+    create=pet.id.apply(lambda id: f"touch \"{id}.txt\""),
+    delete=pet.id.apply(lambda id: f"rm \"{id}.txt\""),
+    # Replace the resource, i.e., re-run the 'create' command,
+    # when the pet resource changes.
+    triggers=[pet.id])
+```
+
+{{% /choosable %}}
+
+{{% choosable language go %}}
+
+```go
+package main
+
+import (
+	"fmt"
+
+	"github.com/pulumi/pulumi-command/sdk/go/command/local"
+	"github.com/pulumi/pulumi-random/sdk/v4/go/random"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+)
+
+func main() {
+	pulumi.Run(func(ctx *pulumi.Context) error {
+		pet, err := random.NewRandomPet(ctx, "pet", nil)
+		if err != nil {
+			return err
+		}
+		_, err = local.NewCommand(ctx, "petFileCommand", &local.CommandArgs{
+			Create: pulumi.String(fmt.Sprintf("touch \"%v.txt\"", pet.ID())),
+			Delete: pulumi.String(fmt.Sprintf("rm \"%v.txt\"", pet.ID())),
+			// Replace the resource, i.e., re-run the 'create' command,
+			// when the pet resource changes.
+			Triggers: pulumi.Array{ pet.ID() },
+		})
+		if err != nil {
+			return err
+		}
+		return nil
+	})
+}
+```
+
+{{% /choosable %}}
+
+{{% choosable language csharp %}}
+
+```csharp
+using Pulumi;
+using Command = Pulumi.Command;
+using Random = Pulumi.Random;
+
+return await Deployment.RunAsync(() =>
+{
+    var pet = new Random.RandomPet("pet");
+
+    var petFileCommand = new Command.Local.Command("petFileCommand", new()
+    {
+        Create = pet.Id.Apply(id => $"touch \"{id}.txt\""),
+        Delete = pet.Id.Apply(id => $"rm \"{id}.txt\""),
+        Triggers = new[]
+        {
+            pet.Id,
+        },
+    });
+});
+```
+
+{{% /choosable %}}
+
+{{< /chooser >}}
+
+Rather than simply putting a “v1” label on the existing provider, we used this opportunity to enhance it with some previously requested capabilities.
+
+- The former `CopyFile` resource was renamed to `CopyToRemote` and can now copy whole directories in addition to individual files.
+- The source of the copy is now a [Pulumi asset or archive](https://www.pulumi.com/docs/concepts/assets-archives/). This provides full interoperability with the Pulumi ecosystem.
+- The use of assets and archives also makes Pulumi run copy operations only if the source has changed.
+- Capturing stdout and stderr of commands can now be switched off, which is useful when they might contain secrets or are very noisy.
+- There are now official examples in the Pulumi examples repository, and examples have also been added to the API documentation in the Pulumi registry.
+- Environment handling for remote commands was fixed and is better documented.
```

Comment:
I would change the order around:

```
- There are now official examples in the Pulumi examples repository, and examples have also been added to the API documentation in the Pulumi registry.
- Capturing stdout and stderr of commands can now be switched off, which is useful when they might contain secrets or are very noisy.
- The former `CopyFile` resource was renamed to `CopyToRemote` and can now copy whole directories in addition to individual files, as well as using [Pulumi asset or archive](https://www.pulumi.com/docs/concepts/assets-archives/) as input.
- Environment handling for remote commands was fixed and is better documented.
```

Can we also add links to more details on each of these?